README.md

```diff
@@ -1,3 +1,4 @@
+![Space screenshot](Space.png "Pe")
```

Comment:
I'm not sure but it would be good if you place the screenshoot below the short description.
Example https://github.com/HaikuArchives/sanity.

Reply:
And, why is the screenshot title "Pe"?  How about "It's full of stars..." :)

Reply:
Sorry for hear that,I forget to change name

Reply:
I changed